BraveRewardsUI/Settings/Tips/Details/TipsDetailViewController.swift

```diff
@@ -182,3 +247,24 @@ extension TipsDetailViewController {
     }
   }
 }
+
+extension TipsDetailViewController {
+  fileprivate func getTipsThisMonth() -> BalanceReportInfo {
+    let components = Calendar(identifier: .gregorian).dateComponents([.month, .year], from: Date())
+    let month = components.month ?? -1
+    let year = components.year ?? 2019
```

Comment:
Don't like defaulting to any given year, should just change this to return `BalanceReportInfo?` and return nil if any info is missing, making it clear we're not receiving a valid balance report instead of just getting an empty one